tests/unit/records/test_schemas.py

```diff
@@ -71,4 +73,42 @@ def test_contributors(app, db, minimal_record):
     minimal_record['contributors'] = [
         {'name': 'test', 'affiliation': 'test', 'type': 'Invalid'}
     ]
-    pytest.raises(ValidationError, Record.create, minimal_record)
+    exc = pytest.raises(ValidationError, Record.create, minimal_record)
+    assert exc.value.message == "'publication' is not of type 'object'"
+
+
+def test_full_json(app, db, full_record):
+    """Test full json."""
+    Record.create(full_record)
+
+
+def test_full_recid(app, db, full_record):
+    """Test recid property."""
+    full_record['recid'] = '12345'
+    exc = pytest.raises(ValidationError, Record.create, full_record)
+    assert exc.value.message == "'publication' is not of type 'object'"
+
+
+def test_full_resource_type(app, db, full_record):
+    """Test resource type."""
+    full_record['resource_type'] = 'publication'
+    exc = pytest.raises(ValidationError, Record.create, full_record)
+    assert exc.value.message == "'publication' is not of type 'object'"
+    full_record['resource_type'] = {'type': 'publication', 'subtype': 'book'}
+    Record.create(full_record)
+
+
+def test_full_contributors(app, db, full_record):
+    """Test contributors."""
+    full_record['contributors'] = [
+       {'affiliation': 'CERN', 'name': ' '', '' ', 'type': 'Other',
+                       'gnd': '', 'orcid': '0000-0002-1825-0097'}
+    ]
+    exc = pytest.raises(ValidationError, Record.create, full_record)
+    assert exc.value.message == "'publication' is not of type 'object'"
```

Comment:
The error:
```
E assert "'publication...ype u'object'" == "'publication'...type 'object'"
E - 'publication' is not of type u'object'
E ? -
E + 'publication' is not of type 'object'
```
 happens because the formatting of this message differs between Python 2 and Python 3, because of the differences between unicode handling (notice the extra 'u' before the 'object'). To fix this one, you need to see if you can somehow render this validation error message, independent of the Python version.